src/upgrades/mod.rs

```diff
@@ -305,7 +313,7 @@ pub fn prepare_upgrade_data_migrations(mode: UpgradeMode, config: Arc<Config>) -
                 let msg = "Cannot upgrade Krill installations from before version 0.6.0. Please upgrade to any version ranging from 0.6.0 to 0.8.1 first, and then upgrade to this version.";
                 error!("{}", msg);
                 Err(PrepareUpgradeError::custom(msg))
-            } else if versions.from < KrillVersion::release(0, 9, 0) {
+            } else if versions.from < KrillVersion::candidate(0, 10, 0, 1) {
```

Comment:
Unrelated to this line, but cannot leave the comment above (due to [#347](https://github.com/github/roadmap/issues/347) not yet being implemented by GitHub):

> Found prepared data for a different krill version, will remove it and start from scratch");

s/krill/Krill/

Reply:
fixed